update.go

```diff
@@ -173,6 +173,10 @@ func checkUpdate(curProfile *Profile, all, force bool) error {
 
 	var images []ImageDef
 	for i := range imagesMap {
+		if i.Image != curProfile.Image {
+			fmt.Printf("skipping update: %s|%s\n", i.Image, i.Platform)
+			continue
+		}
```

Comment:
Alternative is `i.Image != curProfile.Image || !strings.HasSuffix(i.Platform, curProfile.Arch)` which could be better but am not certain that it is impossible for `curProfile.Arch==""` and am being cautious